atc/db/migration/migrations/1522178770_add_job_tags.up.go

```diff
@@ -14,8 +14,8 @@ type GroupConfig struct {
 type GroupConfigs []GroupConfig
 
 type Pipeline struct {
-	ID     int          `json:"id"`
-	Groups GroupConfigs `json:"groups,omitempty"`
+	ID     int
+	Groups GroupConfigs
```

Comment:
Did you mean to remove these?

Reply:
Yep, they don't actually do anything; the object was never used as JSON.